style/components/ThumbnailOption.scss

```diff
@@ -18,6 +18,7 @@
 
   &__thumbnail {
     width: 100%;
+    display: block;
```

Comment:
This removes gap under image if it was placed directly into thumbnail wrapper (currently we add this line in our codebase)